ape.toml.template

```diff
@@ -1,11 +1,19 @@
 [Agent]
-# File paths should be double quoted.
-# Certificate file for the metric servers for prometheus
+# Exporter HTTPS (TLS) configuration
+# HTTPS between Prometheus and Exporter
+
+# Server certificate
 cert_file = "${AGENT_CERT_FILE}"
 
-# Key file for the metric servers for prometheus
+# Private key associated with server certificate
 key_file = "${AGENT_KEY_FILE}"
 
+# Root CA to validate client certificates (for mutual TLS)
+root_ca = "${AGENT_ROOT_CA}"
+
+# Passphrase for encrypted key file
+key_file_passphrase = "${AGENT_KEY_FILE_PASSPHRASE}"
```

Comment:
This is a security hazard. Should only be passed as env variable (or any method that is standard to the industry)

Reply:
I forgot to add comments on the config usage. Yes, it can be passed in following formats. I have added the comments now in the recent commit. Thanks.

```
# Passphrase for encrypted key_file. Supports below formats,
# 1. Passphrase directly                                                      - "<passphrase>"
# 2. Passphrase via file                                                      - "file:<file-that-contains-passphrase>"
# 3. Passphrase via environment variable                                      - "env:<environment-variable-that-holds-passphrase>"
# 4. Passphrase via environment variable containing base64 encoded passphrase - "env-b64:<environment-variable-that-contains-base64-encoded-passphrase>"
# 5. Passphrase in base64 encoded form                                        - "b64:<base64-encoded-passphrase>"
```